docs/hello_nextflow/05_hello_operators.md

```diff
@@ -258,6 +258,7 @@ It should look very familiar, but feel free to run it if you want to satisfy you
 We're going to start by making two changes:
 
 -   Add the `-ERC GVCF` parameter to the GATK HaplotypeCaller command;
+-   Make sure you add a backslash (`\`) at the end of the line before;
```

Comment:
Maybe at this below or outside the list? This makes it a three-line list that we introduce by saying "start by making two changes:"

Reply:
Ops, you're right.